data/json/recipes/recipe_deconstruction.json

```diff
@@ -119,12 +119,7 @@
     "time": 1200,
     "reversible": true,
     "qualities": [ {"id":"HAMMER","level":1} ],
```

Comment:
There is a lot a missing, in part 1 and 2, I don't put this space yet. I just want to make it when I finish it all.